tests/test_event_bus.c

```
@@ -0,0 +1,263 @@
+    ASSERT_EQ_INT(g_call_order[0], 'A');
+
+    obc_event_fire(NULL, "ev2", -1, NULL);
+    ASSERT_EQ_INT(g_call_count, 2);
+    ASSERT_EQ_INT(g_call_order[1], 'B');
+}
+
+TEST(priority_clamp_high)
+{
+    /* priority > 255 is clamped to 255 -- should still subscribe without error */
+    reset_state();
+    int rc = obc_event_subscribe("clamp_hi", handler_a, 1000);
```

Clang (14.0.6)

[warning] variable name 'rc' is too short, expected at least 3 characters

(readability-identifier-length)